app/controllers/sms_controller.rb

```diff
@@ -3,12 +3,15 @@ class SmsController < ApiController
   before_action :ensure_twilio_request
 
   def status_callback
-    sms_message = SmsMessage.find_by(message_sid: params["MessageSid"])
+    sms_message = SmsMessage.find_or_initialize_by(message_sid: params["MessageSid"])
     if sms_message
       status = params["MessageStatus"]
       sms_message.update(
         status: status,
-        error_code: params["ErrorCode"]
+        error_code: params["ErrorCode"],
+        to: params["To"],
```

Comment:
Needed to add to, from and direction to satisfy our non-null requirements. Direction should always be outbound-api for any message we're getting status update callbacks on.